sympy/matrices/tests/test_eigen.py

```diff
@@ -572,3 +572,16 @@ def test_definite():
     assert m.is_positive_definite == True
     assert m.is_positive_semidefinite == True
     assert m.is_indefinite == False
+
+    # test for issue 19547: https://github.com/sympy/sympy/issues/19547
+    m = Matrix([
+        [0, 0, 0],
+        [0, 1, 2],
+        [0, 2, 1]
+    ])
+    assert not m.is_positive_definite, (
+        'The matrix \n\tm = {m}\nIs not positive definite,'
+        ' but m.is_positive_definite == True'.format(m=m))
```

Comment:
I don't think that any debug messages should be added for tests

Reply:
I agree that for these particular tests, the messages do not add much value since the matrix being tested is in the code. Removed them in the most recent commit.